lib/driverProviders/browserStack.ts

```diff
@@ -99,6 +86,12 @@ export class BrowserStack extends DriverProvider {
     this.config_.capabilities['browserstack.key'] = this.config_.browserstackKey;
     this.config_.seleniumAddress = 'http://hub.browserstack.com/wd/hub';
 
+    this.browserstackClient = BrowserstackClient.createAutomateClient({
+      username: this.config_.browserstackUser,
+      password: this.config_.browserstackKey,
+      proxy: this.config_.browserstackProxy
```

Comment:
If this is not set, will it break this code? It would be better to have a test for it,

Reply:
I have added test for browserStack driverProvider's basic functionality check.
It will not break by this.
And tests are now failing in circleci with error "Error: does not exist in logs: WebDriverError: Sauce Labs Authentication Error." and in travis due to some reason. I think there is some issue with Sauce labs credentials, can you please check and review this PR.